CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## [next] 
 - Adding Component#onDestroy
 - Adding Keyboard events API
+- Add onComponentComplete for Animation
```

Comment:
I think here would be onCompleteAnimation.

Also adds a (thanks @diegomgarcia) add the end to mark your contribution 😉

Reply:
I don't believe I've done that ... LOL